src/resources/platform_handlers/pytorch_libtorch/README.md

```diff
@@ -0,0 +1,88 @@
+<!--
+# Copyright 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Serving PyTorch models using Python Backend \[Experimental\]
+
+*NOTE*: This feature is subject to change and removal, and should not
+be used in production.
+
+Starting from 23.08, we are adding an experimental support for loading and
+serving PyTorch models directly without using TorchScript via Python backend.
+The model can be provided within the triton server model repository without
+crafting the `model.py`, and a pre-built Python model [`model.py`](model.py)
+will be used to load and serve the provided PyTorch model.
+
+The model repository structure should look like:
+
+```
+model_repository/
+`-- pytorch_model
+    |-- 1
+    |   |-- model.py
+    |   `-- model.py.pt
+    `-- config.pbtxt
+```
+
+The `model.py` contains the class definition of the PyTorch model. The class
+name needs to match the model name (i.e. `pytorch_model`), and the class should
+extend the
```

Comment:
does this mean that the class name in model.py file should be `class pytorch_model`? Do you have an example model.py to share?

Reply:
I just saw the example you shared in Teams chat. Is there a way to automatically find the model name instead of forcing the user to specify `pytorch_model`? It looks like TorchServe is able to do that.

Reply:
Yes. [Add ability to auto find model class](https://github.com/triton-inference-server/python_backend/pull/282/commits/03893163f551e1fdf11a3d145831eca5f1f192b7)